src/mds/PurgeQueue.h

```diff
@@ -45,11 +50,35 @@ class PurgeItem
   fragtree_t fragtree;
 
   PurgeItem()
-   : action(NONE), ino(0), size(0)
+   : stamp(ceph_clock_now()), pad_size(0), action(NONE), ino(0), size(0)
```

Comment:
stamp(ceph_clock_now()) wastes cpu cycle when reading purge queue item from journal

Reply:
Ok:-) I'll correct that:-).
BTW, I see you removed the last comment, is there anything wrong?